src/components/StarRating/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactComponent as Star } from 'assets/svg/star.svg';
+import { useState } from 'react';
+import styles from './StarRating.module.scss';
+
+interface StarRatingProps {
+  rating: () => void;
```

Comment:
rating 이라는 네이밍을 보면 함수를 기대하기 어려울것 같아요.
onClick 이 더 일반적인 네이밍 인것 같습니다.
```suggestion
  onClick?: () => void;
```

Reply:
네 수정하겠습니다!